CRITICAL - 2025-06-26 00:19:01 --> ParseError: syntax error, unexpected token "echo"
[Method: GET, Route: index]
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\ProT3_38870558\\app\\Controllers\\Home.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Home')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Home')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\Home', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-26 00:19:32 --> ErrorException: Undefined variable $titulo
[Method: GET, Route: quienes_somos]
in APPPATH\Views\front\head_view.php on line 7.
 1 APPPATH\Views\front\head_view.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $titulo', 'C:\\xampp\\htdocs\\ProT3_38870558\\app\\Views\\front\\head_view.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\ProT3_38870558\\app\\Views\\front\\head_view.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('front/head_view', [], true)
 5 APPPATH\Controllers\Home.php(19): view('front/head_view')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->quienes_somos()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-26 00:20:19 --> ErrorException: Undefined variable $titulo
[Method: GET, Route: quienes_somos]
in APPPATH\Views\front\head_view.php on line 7.
 1 APPPATH\Views\front\head_view.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $titulo', 'C:\\xampp\\htdocs\\ProT3_38870558\\app\\Views\\front\\head_view.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\ProT3_38870558\\app\\Views\\front\\head_view.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('front/head_view', [], true)
 5 APPPATH\Controllers\Home.php(19): view('front/head_view')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->quienes_somos()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-26 00:20:24 --> ErrorException: Undefined variable $titulo
[Method: GET, Route: acercade]
in APPPATH\Views\front\head_view.php on line 7.
 1 APPPATH\Views\front\head_view.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $titulo', 'C:\\xampp\\htdocs\\ProT3_38870558\\app\\Views\\front\\head_view.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\ProT3_38870558\\app\\Views\\front\\head_view.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('front/head_view', [], true)
 5 APPPATH\Controllers\Home.php(28): view('front/head_view')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->acercade()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-26 00:20:41 --> ErrorException: Undefined variable $titulo
[Method: GET, Route: quienes_somos]
in APPPATH\Views\front\head_view.php on line 7.
 1 APPPATH\Views\front\head_view.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $titulo', 'C:\\xampp\\htdocs\\ProT3_38870558\\app\\Views\\front\\head_view.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\ProT3_38870558\\app\\Views\\front\\head_view.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('front/head_view', [], true)
 5 APPPATH\Controllers\Home.php(19): view('front/head_view')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->quienes_somos()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-26 00:23:52 --> ParseError: syntax error, unexpected token "echo"
[Method: GET, Route: login]
in APPPATH\Controllers\Home.php on line 11.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\ProT3_38870558\\app\\Controllers\\Home.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Home')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Home')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\Home', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-26 00:25:37 --> ParseError: syntax error, unexpected token "echo"
[Method: GET, Route: login]
in APPPATH\Controllers\Home.php on line 11.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\ProT3_38870558\\app\\Controllers\\Home.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Home')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Home')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\Home', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-26 01:52:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-26 01:52:04 --> Error: Call to undefined function set_value()
[Method: GET, Route: registro]
in APPPATH\Views\back\usuario\registro.php on line 29.
 1 SYSTEMPATH\View\View.php(224): include()
 2 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('back/usuario/registro', [], true)
 4 APPPATH\Controllers\Home.php(39): view('back/usuario/registro')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->registro()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-26 01:53:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-26 01:53:23 --> Error: Call to undefined function set_value()
[Method: GET, Route: registro]
in APPPATH\Views\back\usuario\registro.php on line 29.
 1 SYSTEMPATH\View\View.php(224): include()
 2 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('back/usuario/registro', [], true)
 4 APPPATH\Controllers\Home.php(39): view('back/usuario/registro')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->registro()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-26 01:53:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-26 01:53:26 --> Error: Call to undefined function set_value()
[Method: GET, Route: registro]
in APPPATH\Views\back\usuario\registro.php on line 29.
 1 SYSTEMPATH\View\View.php(224): include()
 2 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('back/usuario/registro', [], true)
 4 APPPATH\Controllers\Home.php(39): view('back/usuario/registro')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->registro()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-26 01:56:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-26 01:56:10 --> Error: Call to undefined function set_value()
[Method: GET, Route: registro]
in APPPATH\Views\back\usuario\registro.php on line 29.
 1 SYSTEMPATH\View\View.php(224): include()
 2 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('back/usuario/registro', [], true)
 4 APPPATH\Controllers\Home.php(39): view('back/usuario/registro')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->registro()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-26 01:59:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:00:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:10:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:21:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:21:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:21:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:22:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:26:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:27:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:27:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:27:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:28:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:37:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:37:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:43:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:43:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-26 02:43:35 --> ErrorException: unlink(C:\xampp\htdocs\ProT3_38870558\writable\debugbar/debugbar_1750896037.031553.json): No such file or directory
[Method: GET, Route: /]
in SYSTEMPATH\Debug\Toolbar\Collectors\History.php on line 82.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(C:\\xampp\\htdocs\\ProT3_38870558\\writable\\debugbar/debugbar_1750896037.031553.json): No such file or directory', 'C:\\xampp\\htdocs\\ProT3_38870558\\system\\Debug\\Toolbar\\Collectors\\History.php', 82)
 2 SYSTEMPATH\Debug\Toolbar\Collectors\History.php(82): unlink('C:\\xampp\\htdocs\\ProT3_38870558\\writable\\debugbar/debugbar_1750896037.031553.json')
 3 SYSTEMPATH\Debug\Toolbar.php(517): CodeIgniter\Debug\Toolbar\Collectors\History->setFiles('1750905814.336308', 20)
 4 SYSTEMPATH\Debug\Toolbar.php(496): CodeIgniter\Debug\Toolbar->format([...], 'html')
 5 APPPATH\Config\Events.php(47): CodeIgniter\Debug\Toolbar->respond()
 6 SYSTEMPATH\Events\Events.php(155): CodeIgniter\Events\Events::Config\{closure}()
 7 SYSTEMPATH\CodeIgniter.php(339): CodeIgniter\Events\Events::trigger('pre_system')
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-26 02:43:56 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\ProT3_38870558\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\ProT3_38870558\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'argota_rodrigo', 3306, '', 0)
#1 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ProT3_38870558\system\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ProT3_38870558\system\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('rodrigoargota@g...', 'usuarios.email', Array)
#6 C:\xampp\htdocs\ProT3_38870558\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('rodrigoargota@g...', 'usuarios.email', Array, NULL, 'email')
#7 C:\xampp\htdocs\ProT3_38870558\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'rodrigoargota@g...', Array, Array, 'email')
#8 C:\xampp\htdocs\ProT3_38870558\system\Controller.php(138): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\ProT3_38870558\app\Controllers\usuario_controller.php(37): CodeIgniter\Controller->validate(Array)
#10 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(933): App\Controllers\usuario_controller->FormValidation()
#11 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#12 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\ProT3_38870558\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\ProT3_38870558\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#15 C:\xampp\htdocs\ProT3_38870558\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\ProT3_38870558\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ProT3_38870558\system\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ProT3_38870558\system\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('rodrigoargota@g...', 'usuarios.email', Array)
#5 C:\xampp\htdocs\ProT3_38870558\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('rodrigoargota@g...', 'usuarios.email', Array, NULL, 'email')
#6 C:\xampp\htdocs\ProT3_38870558\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'rodrigoargota@g...', Array, Array, 'email')
#7 C:\xampp\htdocs\ProT3_38870558\system\Controller.php(138): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ProT3_38870558\app\Controllers\usuario_controller.php(37): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(933): App\Controllers\usuario_controller->FormValidation()
#10 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ProT3_38870558\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ProT3_38870558\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\ProT3_38870558\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-26 02:43:56 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: POST, Route: usuario_controller/FormValidation]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('rodrigoargota@gmail.com', 'usuarios.email', [...])
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('rodrigoargota@gmail.com', 'usuarios.email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'rodrigoargota@gmail.com', [...], [...], 'email')
 7 SYSTEMPATH\Controller.php(138): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\usuario_controller.php(37): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->FormValidation()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-26 02:44:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:48:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-26 02:49:02 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\ProT3_38870558\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\ProT3_38870558\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'argota_rodrigo', 3306, '', 0)
#1 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ProT3_38870558\system\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ProT3_38870558\system\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('rodrigoargota@g...', 'usuarios.email', Array)
#6 C:\xampp\htdocs\ProT3_38870558\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('rodrigoargota@g...', 'usuarios.email', Array, NULL, 'email')
#7 C:\xampp\htdocs\ProT3_38870558\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'rodrigoargota@g...', Array, Array, 'email')
#8 C:\xampp\htdocs\ProT3_38870558\system\Controller.php(138): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\ProT3_38870558\app\Controllers\usuario_controller.php(37): CodeIgniter\Controller->validate(Array)
#10 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(933): App\Controllers\usuario_controller->FormValidation()
#11 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#12 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\ProT3_38870558\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\ProT3_38870558\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#15 C:\xampp\htdocs\ProT3_38870558\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\ProT3_38870558\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ProT3_38870558\system\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ProT3_38870558\system\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('rodrigoargota@g...', 'usuarios.email', Array)
#5 C:\xampp\htdocs\ProT3_38870558\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('rodrigoargota@g...', 'usuarios.email', Array, NULL, 'email')
#6 C:\xampp\htdocs\ProT3_38870558\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'rodrigoargota@g...', Array, Array, 'email')
#7 C:\xampp\htdocs\ProT3_38870558\system\Controller.php(138): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ProT3_38870558\app\Controllers\usuario_controller.php(37): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(933): App\Controllers\usuario_controller->FormValidation()
#10 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ProT3_38870558\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ProT3_38870558\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\ProT3_38870558\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-26 02:49:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: POST, Route: usuario_controller/FormValidation]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('rodrigoargota@gmail.com', 'usuarios.email', [...])
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('rodrigoargota@gmail.com', 'usuarios.email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'rodrigoargota@gmail.com', [...], [...], 'email')
 7 SYSTEMPATH\Controller.php(138): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\usuario_controller.php(37): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->FormValidation()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-26 02:49:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-26 02:49:23 --> Error: Class "App\Models\usuario_models" not found
[Method: POST, Route: usuario_controller/login]
in APPPATH\Controllers\usuario_controller.php on line 104.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->login()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-26 02:54:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-26 02:56:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-26 02:57:34 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\ProT3_38870558\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\ProT3_38870558\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'argota_rodrigo', 3306, '', 0)
#1 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ProT3_38870558\system\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ProT3_38870558\system\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('rodrigoargota@g...', 'usuarios.email', Array)
#6 C:\xampp\htdocs\ProT3_38870558\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('rodrigoargota@g...', 'usuarios.email', Array, NULL, 'email')
#7 C:\xampp\htdocs\ProT3_38870558\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'rodrigoargota@g...', Array, Array, 'email')
#8 C:\xampp\htdocs\ProT3_38870558\system\Controller.php(138): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\ProT3_38870558\app\Controllers\usuario_controller.php(37): CodeIgniter\Controller->validate(Array)
#10 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(933): App\Controllers\usuario_controller->FormValidation()
#11 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#12 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\ProT3_38870558\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\ProT3_38870558\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#15 C:\xampp\htdocs\ProT3_38870558\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\ProT3_38870558\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ProT3_38870558\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ProT3_38870558\system\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ProT3_38870558\system\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('rodrigoargota@g...', 'usuarios.email', Array)
#5 C:\xampp\htdocs\ProT3_38870558\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('rodrigoargota@g...', 'usuarios.email', Array, NULL, 'email')
#6 C:\xampp\htdocs\ProT3_38870558\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'rodrigoargota@g...', Array, Array, 'email')
#7 C:\xampp\htdocs\ProT3_38870558\system\Controller.php(138): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ProT3_38870558\app\Controllers\usuario_controller.php(37): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(933): App\Controllers\usuario_controller->FormValidation()
#10 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\ProT3_38870558\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ProT3_38870558\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ProT3_38870558\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\ProT3_38870558\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-26 02:57:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: POST, Route: usuario_controller/FormValidation]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('rodrigoargota@gmail.com', 'usuarios.email', [...])
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('rodrigoargota@gmail.com', 'usuarios.email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'rodrigoargota@gmail.com', [...], [...], 'email')
 7 SYSTEMPATH\Controller.php(138): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\usuario_controller.php(37): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->FormValidation()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-26 02:57:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-26 02:57:53 --> Error: Class "App\Models\usuario_models" not found
[Method: POST, Route: usuario_controller/login]
in APPPATH\Controllers\usuario_controller.php on line 104.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->login()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
